wasmbinding/test/store_run_test.go

```diff
@@ -39,17 +40,21 @@ func storeCodeViaProposal(t *testing.T, ctx sdk.Context, quicksilverApp *app.Qui
 	wasmCode, err := os.ReadFile("../testdata/hackatom.wasm")
 	require.NoError(t, err)
 
-	src := types.StoreCodeProposalFixture(func(p *types.StoreCodeProposal) {
+	checksum, err := hex.DecodeString("5686AE32B4CC3811109CD4B37A80B8A99B9FF35323AA4FA0485174A8DF1E5A77")
```

Comment:
why do we have this string @sontrinh16

Reply:
[https://github.com/CosmWasm/wasmd/blob/main/x/wasm/types/test_fixtures.go#L198](url) we should create a checksum base on the address we use